### DISCUSSION TITLE
How to attach second database in DuckDB?

### DISCUSSION THREAD

**Discussion by Admolly** [duckdb/duckdb#3542] [General]
I see the following PRAGMA in the documentation (https://duckdb.org/docs/sql/pragmas): 

-- list all databases, usually one
```PRAGMA database_list;```

My question is, how do I actually go about attaching a second database in DuckDB?

In SQLite, I believe the command is: 

```ATTACH DATABASE file_name AS database_name;```

Executing this in DuckDB throws an error, so how do we do it?

**Comment by Mytherin** [comment]
The attach functionality is not yet supported (see #1934). We support the `PRAGMA database_list` command for compatibility with SQLite programs that use this command; currently it is not possible to attach more than one database.

**Comment by Admolly** [comment]
Understood.  I agree with the previous comment, being able to attach to multiple DBs at the same time would be very helpful!

**Comment by cookiejest** [comment]
Hi is this valid or is it now possible to attach more than one?

**Comment by dberardo-com** [comment]
is this feature also in discussion for the postgres_attach command?

would that work in case the 2,3,... multliple attached databases have the same table names ? (use case --> multi-tenant environment)
